core.ts

```diff
@@ -220,7 +220,9 @@ class Poll<
 	}
 }
 
-const transformGetResult = (params: PinejsClientCoreFactory.ParamsObj) => {
+export const transformGetResult = (
```

Comment:
I think I would prefer to put this as a protected method on the PinejsClientCore class rather than exporting it directly like this

Reply:
Will do. That's what I was thinking as well, but thought that you might not like the extra 4 bytes on the prototype :)